### ISSUE TITLE
Icon positioning

### ISSUE THREAD

**Issue by dricholm** [westracer/fontify#3]
Hello,

I have an issue/feature request about icon positioning. I will try to describe the situation with some pictures.

I have 3 SVG files, each one represents a bar in a progress/signal display like icon:
![image](https://user-images.githubusercontent.com/32329112/91697377-4e8e4580-eb71-11ea-8dff-65902cb96a25.png)
They are separate because the colors need to be different depending on some circumstances, so I plan to display each in a Stack so I can color them separately. Every SVG has 128x128 width + height and view box.

Instead of building/launching the app I use this website to inspect the glyphs inside the font: [Torinak](https://torinak.com/font/lsfont.html).
When I build the font with fontify it will resize and center every path:
![image](https://user-images.githubusercontent.com/32329112/91697828-0c193880-eb72-11ea-8ce1-28cef9187f38.png)
This is because of the normalize option I think, so when I try it with normalize: false this is the result:
![image](https://user-images.githubusercontent.com/32329112/91697899-2a7f3400-eb72-11ea-9e76-79afa6813bb7.png)
As you can see in this case the icons are not in the correct position.

When I try to import the same files in [FlutterIcon](https://www.fluttericon.com/) the result is what I am aiming for:
![image](https://user-images.githubusercontent.com/32329112/91698177-9366ac00-eb72-11ea-8f9a-54a649a7fa67.png)

Would it be possible to have the same output as with FlutterIcon? Do you think this could be a bug or can this feature be implemented?
I would prefer using Fontify if this can be accomplished as the project would not need an external website to convert the icons.

Thank you!

**Comment by westracer** [comment]
Hello @dricholm. Yes, you're right, it looks like a problem related to disabled normalization. Though normalized glyphs look fine to me. I'll look into it. Thank you.

**Comment by westracer** [comment]
Fixed and released in 0.0.3. Reopen the issue if the problem is still there.

**Comment by dricholm** [comment]
Thank you for the update @westracer !
I just tried it, seems to have gotten a lot better. However it seems every icon is moved towards the top part a bit with the bottom part being empty. Let me show you some screenshots of what I mean. Here is a box icon where it can be seen clearly.

The original SVG file inspected in Chrome:
![svg_original](https://user-images.githubusercontent.com/32329112/92224822-a5ae5600-eea2-11ea-9400-93686acdb5f4.png)
Padding around the box is even.

The generated font inspected in the above mentioned website:
![svg_fontify](https://user-images.githubusercontent.com/32329112/92224991-e1e1b680-eea2-11ea-8970-47a652158e1b.png)

Also a screenshot from the app with debug painting:
![flutter_54](https://user-images.githubusercontent.com/32329112/92225631-c9be6700-eea3-11ea-8913-d27ac841a4d3.png)
The top left X icon is the default Material Icon, other icons are generated with Fontify. Those icons are also a bit outside the border drawn by debug painting.

It seems horizontally they look fine. Maybe there is some top padding missing or something?

**Comment by westracer** [comment]
@dricholm I'll check it out. Could you, please, provide a few SVG icons, so it would be easier for me to detect the problem? Thanks.

**Comment by dricholm** [comment]
I just tried it with normalize: true and it works okay.
The above text only applies in case normalize is false. I don't know if that is the intended behavior.
Do you think this should still be an issue? It should work fine for my case if normalize is true.

Edit: Sorry, forgot about centering in case normalize is true. So it should still be an issue.
Here are some icons:

Stop:
```svg
<svg width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
  <path d="M107 16a5 5 0 0 1 5 5v86a5 5 0 0 1-5 5H21a5 5 0 0 1-5-5V21a5 5 0 0 1 5-5h86z"/>
</svg>
```

Play:
```svg
<svg width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
  <path d="M104.762 54.538L39.709 17.02C32.729 12.99 24 18.026 24 26.08v75.04c0 8.055 8.73 13.091 15.709 9.062l65.053-37.518c6.984-4.03 6.984-14.098 0-18.127"/>
</svg>
```

Let me know if you need anything else. Thank you!

**Comment by westracer** [comment]
I've released another fix in 0.1.1. Waiting for your feedback, thanks.

**Comment by dricholm** [comment]
I just tested it, works perfectly.
Thank you @westracer !
